_rules/bypass-blocks-cf77f2.md

```diff
@@ -0,0 +1,554 @@
+---
+id: cf77f2
+name: Bypass Blocks of Repeated Content
+rule_type: composite
+description: |
+  This rule checks that each page has a mechanism to bypass repeated blocks of content.
+accessibility_requirements:
+  wcag20:2.4.1: # Bypass Blocks (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G1: # Adding a link at the top of each page that goes directly to the main content area
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G123: # Adding a link at the beginning of a block of repeated content to go to the end of the block
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G124: # Adding links at the top of the page to each area of the content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:H69: # Providing heading elements at the beginning of each section of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:SCR28: # Using an expandable and collapsible menu to bypass block of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_rules:
+  - 047fe0
+  - b40fd1
+  - 3e12e1
+  - ye5d6e
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _Romance of the Three Kingdoms_ by Luo Guanzhong, translation by Charles Henry Brewitt-Taylor (Tuttle Publishing, 1925, ISBN 9780804834674)
+    - _Three Kingdoms_ by Luo Guanzhong, translation by Moss Roberts (Foreign Language Press, 1976, ISBN 7-119-00590-1)
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [_Block of repeated content is collapsible_][block collapsible]; or
+- [_Document has heading for non-repeated content_][document has heading for main]; or
+- [_Document has a main landmark_][document has main]; or
+- [_Document has an instrument to move focus to non-repeated content_][document has instrument to main].
+
+## Assumptions
```

Comment:
This rule is assuming that pages with more than one block of repeating content, the mechanism to bypass each block will be the same. If there are two blocks, one that can be bypassed with a skiplink, and another with a heading, this rule would fail.